glooshot/docs/tutorial/bookinfo_tutorial.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Bookinfo
+menuTitle: Bookinfo
+weight: 1
+---
+
+## Bookinfo Tutorial
+
+This tutorial will show you how to use Glooshot to apply chaos experiments to a simple service mesh app.
+We will use a slight modification of the familiar bookinfo app from Istio's
+[sample app repo](https://github.com/istio/istio/tree/master/samples/bookinfo). We have modified the reviews service to
+include a vulnerability that can lead to cascading failure. We will use Glooshot to detect this weakness.
+
+
+#### The Goal
+
+Services should be resilient to system outages. In this example, we show how to detect cascading failures: failures
```

Comment:
I might rephrase this one a bit. "Services should be built to be resilient when dependencies are unavailable in order to avoid cascading failures. In this example, ......"